docs/source/dataprofiler.profilers.rst

```diff
@@ -22,6 +22,7 @@ Modules
    dataprofiler.profilers.graph_profiler
    dataprofiler.profilers.histogram_utils
    dataprofiler.profilers.int_column_profile
+   dataprofiler.profilers.json_decoder
```

Comment:
do we want the decoder stuff? It's not completely functional right now. Thoughts?

Reply:
I think that might be next release. Just because we have a decent amount work left on it

Reply:
Agree

Reply:
will fix

Reply:
removing decoder